### ISSUE TITLE
Any amount in excess of 999.00 is truncated

### ISSUE THREAD

**Issue by avma** [moneymanagerex/moneymanagerex#3645]
MMEX version:
 - [X] 1.6.0 alph 1

Note: bug reporters are expected to have verified the bug still exists
either in the last stable version of MMEX or on updated development code
(master branch).

Operating System:
  - [X] Linux Mint 20.1

# Description of the bug
In the transaction dialog, any amount exceeding 999.00 is cut, for example: if you enter 1,581.00 it will be shortened to 1.00. The same thing happens when copying an existing transaction, when the amount is greater than 999.00. I tried to copy an existing movement for 5,300.00 and it was truncated to 5.00.
Please describe your issue with details.
Add screenshot or other files if needed.

# Reproduction

Is the bug reproducible? 
 - [X] Always 
 - [ ] Randomly 
 - [ ] Happened only once

Reproduction steps:

1. open transaction Dialog
2. enter amount = 1000.00
3. post

…

Expected result:1,000.00

Actual result:1.00

# Additional information
```
Money Manager Ex
Version: 1.6.0-Alpha.1 64-bit
• Build on May 12 2021 14:50:52
• Database version: 7 • (aes128cbc)
MMEX is using the following support products:
• wxWidgets 3.1.5 (wxGTK 3.24)
• wxSQLite3 4.6.0 (SQLite 3.31.1) • RapidJSON 1.1.0
• Lua 5.3.3
• lunasvg 
• libcurl/7.68.0 OpenSSL/1.1.1f zlib/1.2.11 brotli/1.0.7 libidn2/2.2.0 libpsl/0.21.0 (+libidn2/2.2.0) libssh/0.9.3/openssl/zlib nghttp2/1.40.0 librtmp/2.3
• gettext 0.19.8.1
• apexcharts.js
Build using:
• CMake 3.16.3
• GNU Make 4.2.1
• GCC 9.3.0 • Linuxmint 20.1 'ulyssa'
Running on:
• Linux Mint 20.1 "ulyssa"
• Linux 5.8.0-59-generic x86_64
• Hebrew (UTF-8)
• 1920x1080 24bit 192x192ppi 
```

If you have a backtrace for a crash or a warning, paste it here.

**Comment by tactilis** [comment]
Duplicate of #3626
